Adhan.d.ts

```diff
@@ -134,6 +136,7 @@ export enum Prayer {
   Sunrise,
   Dhuhr,
   Asr,
+  Sunset,
```

Comment:
we can add sunset to the PrayerTimes object but I don't want to change the `Prayer` enum.